Formula/cpmtools.rb

```diff
@@ -22,21 +32,25 @@ def install
   end
 
   test do
+    # libdsk support needs an image type
+    libdsk_type = []
+    libdsk_type += %W[-T logical] if build.with? "libdsk"
```

Comment:
If it's optional you can skip making any `test` changes as they will never be run by CI.